### PR TITLE
Create Readme.MD for KanBERTo

### PR DESCRIPTION
**Pull request by Naveenkhasyap**
KanBERTo language model readme for Kannada language. which I trained by following your blog.

Fixes #{issue number}

### CODE REVIEW

**Comment by Naveenkhasyap** [comment]
please let me know how to mention language code (kn) also so that it will be easy to filter on website.

File: model_cards/Naveen-k/KanBERTo/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Welcome to KanBERTo (ಕನ್ಬರ್ಟೋ)
```
Comment (julien-c) [inline model_cards/Naveen-k/KanBERTo/README.md:1]:
```suggestion
---
language: kn
---

# Welcome to KanBERTo (ಕನ್ಬರ್ಟೋ)
```

File: model_cards/Naveen-k/KanBERTo/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Model Description
+ 
+> This is a small language model for [Kannada](https://en.wikipedia.org/wiki/Kannada) language with 1M data samples taken from
+  [OSCAR page](https://traces1.inria.fr/oscar/files/compressed-orig/kn.txt.gz)
+
+## Training params 
+
+- **Dataset** - 1M data samples are used to train this model from OSCAR page(https://traces1.inria.fr/oscar/) eventhough data set is of 1.7 GB due to resource constraint to train 
+I have picked only 1M data from the total 1.7GB data set. If you are interested in collaboration and have computational resources to train on you are most welcome to do so.
```
Comment (julien-c) [inline model_cards/Naveen-k/KanBERTo/README.md:11]: have you tried running on Google Colab? and/or Colab Pro?

**Comment by julien-c** [comment]
> please let me know how to mention language code (kn) also so that it will be easy to filter on website.

There you go!

Thanks for sharing

**Comment by julien-c** [comment]
https://huggingface.co/models?filter=kn

File: model_cards/Naveen-k/KanBERTo/README.md (hunk 2 above)
Comment (Naveenkhasyap) [reply model_cards/Naveen-k/KanBERTo/README.md:15]: Hi @julien-c , Yes i have tried training with the same data set with smaller size and was able to train model from it.

**Comment by Naveenkhasyap** [comment]
> 
> 
> https://huggingface.co/models?filter=kn

Thanks a lot @julien-c 👍 .

**Pull request merged**